.github/workflows/docker.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        backend: [jax]
+        backend: [jax, torch]
```

Comment:
The PyTorch images might be too big to fit on the runners, so this might need to get dropped with manual builds and deploys.